deps.Dockerfile

```diff
@@ -11,11 +11,13 @@ RUN ["go", "build", "-ldflags", "-s -w", "."]
 RUN ["upx", "entrypoint"]
 
 
-FROM alpine as empty
+FROM alpine as base
 RUN ["mkdir", "/empty"]
+COPY rootfs/ /rootfs
+RUN ["chmod", "-R", "a+rX", "/rootfs"]
```

Comment:
Why X, not x?

Reply:
Otherwise files will become executable even though they shouldn't:
> execute/search only if the file is a directory or already has execute permission for some user (X)

Reply:
OK.

Reply:
Wait, I actually wanted to push a version where the mode is `u=rwX,go=rX`. What do you think about this instead? This would ensure exact permissions but should make no difference for any sane use-case.

Reply:
Sound good.